lib/avo/fields/area_field.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Avo
+  module Fields
+    class AreaField < BaseField
+      attr_reader :geometry
+      attr_reader :options
+      attr_reader :datapoint_options
+      def initialize(id, **args, &block)
+        hide_on :index
+        super(id, **args, &block)
+
+        @geometry = args[:geometry].present? ? args[:geometry] : :polygon # Defaults to `polygon`; Possible values: `:polygon`, `:multi_polygon`
```

Comment:
You might be able to make codeclimate happier here by using `presence`:

```suggestion
        @geometry = args[:geometry].presence || :polygon # Defaults to `polygon`; Possible values: `:polygon`, `:multi_polygon`
```